src/main/java/org/jenkinsci/plugins/KeycloakAccess.java

```diff
@@ -50,16 +50,21 @@ public KeycloakAccess( KeycloakDeployment keycloakDeployment ) {
 	public UserDetails loadUserByUsername( String username ) throws UsernameNotFoundException, DataAccessException {
 		LOGGER.finer( "Requested User Details for: " + username );
 		try {
-			if (!cache.isInvalidUser( username )) {
+			if (cache.isEnabled() && !cache.isInvalidUser( username )) {

```

Comment:
So if the cache isn't enabled, this will also fail (i.e. it won't let user attributes to be retrieved unless the cache is enabled? Is that intended?

Reply:
No.. Good catch.. I'll fix that.

Reply:
I fixed the logic.